starter-code/app.js

```diff
@@ -54,5 +55,7 @@ app.locals.title = 'Express - Generated with IronGenerator';
 const index = require('./routes/index');
 app.use('/', index);
 
+app.use('/', require('./routes/celebrities.js')
+app.use('/', require('./routes/movies.js')
```

Comment:
In these two lines, I would advise you to always add the name on the first argument. Its easier to read what those routes are, theres no chance for conflicts and easier for you to debug.
```js
app.use('/celebrities', ....
```
If you just add the slash, literally any of those files might throw an error. If you silo the files into their own routes, you just have to look into one place